stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -475,6 +475,22 @@ extension AsyncStream: @unchecked Sendable where Element: Sendable { }
 @available(SwiftStdlib 5.1, *)
 extension AsyncStream.Continuation.YieldResult: Sendable where Element: Sendable { }
 
+@available(SwiftStdlib 6.2, *)
+extension AsyncStream.Continuation: Hashable {
+  @available(SwiftStdlib 6.2, *)
```

Comment:
Should these all be `@inlinable`?

Reply:
Here's the followup: 

- main: https://github.com/swiftlang/swift/pull/81126
- 6.2: https://github.com/swiftlang/swift/pull/81127